src/map/attackround.cpp

```diff
@@ -199,12 +199,23 @@ void CAttackRound::DeleteAttackSwing()
 void CAttackRound::CreateAttacks(CItemWeapon* PWeapon, PHYSICAL_ATTACK_DIRECTION direction)
 {
     uint8 num = 1;
+    
+    bool isPC = m_attacker->objtype == TYPE_PC;
 
     // Checking the players weapon hit count
     if (PWeapon->getReqLvl() <= m_attacker->GetMLevel())
     {
         num = PWeapon->getHitCount();
     }
+    
+    // If the attacker is a mob, check to see if it has any special mutli-hit capabilties
+    if (!isPC)
```

Comment:
you can either dynamic_cast it to a mobentity or do that

I think dynamic_cast is slower but if anything new derives from mobentity it'll already work without having to change that line..